test/functional/mempool_reorg.py

```
@@ -22,8 +33,84 @@ def set_test_params(self):
             []
         ]
 
+    def test_reorg_relay(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_reorg_relay`

Add return type annotation: `None`

(ANN201)